pkg/actions/cluster/unowned.go

```diff
@@ -81,8 +86,13 @@ func (c *UnownedCluster) Delete(waitInterval time.Duration, wait, force bool) er
 			return err
 		}
 
-		if err := drainAllNodegroups(c.cfg, c.ctl, c.stackManager, clientSet, allStacks); err != nil {
-			return err
+		nodeGroupManager := c.newNodeGroupManager(c.cfg, c.ctl, clientSet)
+		if err := drainAllNodeGroups(c.cfg, c.ctl, clientSet, allStacks, disableNodegroupEviction, nodeGroupManager, attemptVpcCniDeletion); err != nil {
+			if force {
+				logger.Warning("an error occurred during nodegroups draining, force=true so proceeding with deletion: %q", err.Error())
+			} else {
+				return err
+			}
```

Comment:
```suggestion
			if !force {
			    return err
			}
			logger.Warning("an error occurred during nodegroups draining, force=true so proceeding with deletion: %q", err.Error())
```

It's a bummer that this has to be repeated. :/ ¯\\\_(ツ)\_\/¯

Reply:
Done